dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/SectionControl.mo

```diff
@@ -0,0 +1,37 @@
+within Dynawo.NonElectrical.Blocks.NonLinear;
```

Comment:
One general coding style comment :
- Copyright missing.
- Indentation and spaces between sections to review